src/components/UnorderedList.tsx

```diff
@@ -1,22 +1,17 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import {View} from 'react-native';
-import _ from 'underscore';
 import styles from '@styles/styles';
 import Text from './Text';
 
-const propTypes = {
+type UnorderedListProps = {
     /** An array of strings to display as an unordered list */
-    items: PropTypes.arrayOf(PropTypes.string),
-};
-const defaultProps = {
-    items: [],
+    items: string[];
 };
 
-function UnorderedList(props) {
+function UnorderedList({items = []}: UnorderedListProps) {
```

Comment:
@JKobrynski Do we need a default value if items are required?

Reply:
My bad, it should not be required